.vscode/extensions.json

```diff
@@ -7,7 +7,8 @@
     "rust-lang.rust-analyzer",
     "ms-azuretools.vscode-docker",
     "manucorporat.vermoji",
-    "vadimcn.vscode-lldb"
+    "vadimcn.vscode-lldb",
+    "streetsidesoftware.code-spell-checker"
```

Comment:
Is this intentional?

Reply:
yes, I wrote it in the description of the PR, this extension spot grammatical errors, and it even supports a range of cases so it's useful even when writing actual code and not  just documentation